yaml/pois.yaml

```diff
@@ -951,15 +965,6 @@ filters:
     output:
       <<: *output_properties
       kind: battlefield
-  # park w/ US Parks Service operator
```

Comment:
Was this a catch-all for any leisure or landuse park before, but with a bad comment?

Reply:
Yes, on line 432 in this diff.